src/exporter.js

```diff
@@ -52,7 +44,7 @@ function exporter (hash, dagService, options, callback) {
         rs.push(unmarshaledData.data)
         rs.push(null)
       }
-      ee.emit('file', { stream: rs, path: name, dir: dir })
+      this.push({ stream: rs, path: name, dir: dir })
```

Comment:
this was being used for the js-ipfs cli get command. But things have changed so it may not be needed, testing now